assets/scss/templates/_big-data.scss

```diff
@@ -0,0 +1,94 @@
+.dp-wrapper-bd {
+  position: relative;
+  margin: $dp-spaces--lv0;
+
+  .dp-bd-box {
+    position: relative;
+  }
+
+  .dp-button-group {
+    position: absolute;
+    right: 0;
+    height: 40px;
+    top: 22px;
```

Comment:
I have an issue here 
![image](https://user-images.githubusercontent.com/2439363/107548404-5e449800-6bad-11eb-92b4-b702b7ecc08f.png)

For this to work correctly in old doppler I need 60px at top, can you help me with this?

Reply:
Old doppler inputs here in this page are 42px tall, here all styles were overwritten, could you add a class to skip input styles? like `old-input` or perhaps `ignore-library` that allows to have the original styles only. You see here the difference is a lot.  
![image](https://user-images.githubusercontent.com/2439363/107550094-61d91e80-6baf-11eb-92e6-428b3614c471.png)

Reply:
It also breaks the next label and error message tooltip 😕

Reply:
![image](https://user-images.githubusercontent.com/10213170/107550264-9220bd00-6baf-11eb-93f1-81b99e85442a.png)
Text format is different too.

Reply:
I think I better take care of this issues in old css, to avoid dirtying the library 😅

Reply:
I can lend you my classes @lucianaDilizio 
![image](https://user-images.githubusercontent.com/2439363/107555111-8afcad80-6bb5-11eb-9b07-d16cff942498.png)